specification/marketplacecatalog/data-plane/Search/stable/2022-02-02/search.json

```diff
@@ -0,0 +1,1065 @@
+        "isMicrosoftProduct": {
+          "type": "boolean"
+        },
+        "isPreview": {
+          "type": "boolean"
+        },
+        "language": {
+          "type": "string"
+        },
+        "legacyId": {
+          "type": "string"
+        },
+        "offerType": {
+          "type": "string"
+        },
+        "operatingSystems": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        },
+        "plans": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/PlanDto"
+          },
+          "x-ms-identifiers": [
+            "legacyPlanId"
+          ]
+        },
+        "pricingTypes": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        },
+        "publisherDisplayName": {
+          "type": "string"
+        },
+        "summary": {
+          "type": "string"
+        },
+        "vmGenerations": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        },
+        "vmImageTypes": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        },
+        "vmSecurityTypes": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        },
+        "smallIconUri": {
+          "type": "string"
+        },
+        "mediumIconUri": {
+          "type": "string"
+        },
+        "largeIconUri": {
+          "type": "string"
+        },
+        "wideIconUri": {
+          "type": "string"
+        },
+        "heroIconUri": {
+          "type": "string"
+        }
+      }
+    },
+    "PlanDto": {
+      "type": "object",
+      "properties": {
+        "cspState": {
+          "type": "string"
+        },
+        "legacyPlanId": {
+          "type": "string"
+        },
+        "displayName": {
+          "type": "string"
+        },
+        "architectureType": {
+          "type": "string"
+        },
+        "metadata": {
+          "$ref": "#/definitions/PlanMetadataDto"
+        },
+        "operatingSystem": {
+          "$ref": "#/definitions/OperatingSystemDto"
+        }
+      }
+    },
+    "PlanMetadataDto": {
+      "type": "object",
+      "properties": {
+        "generation": {
+          "type": "string"
+        }
+      }
+    },
+    "OperatingSystemDto": {
+      "type": "object",
+      "properties": {
+        "family": {
+          "type": "string"
+        },
+        "type": {
+          "type": "string"
+        },
+        "name": {
+          "type": "string"
+        }
+      }
+    }
+  },
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
```

Comment:
Oauth2 scope names should have the form: https://<audience>/<permission>

Reply:
@mikekistler  can you please explain? I'm looking at other examples and I see they are the same as I defined.
If I dont have a need for security since its public API, how can I set some basic or something like that?